holopy/inference/tests/test_result.py

```diff
@@ -71,36 +73,77 @@ def test_properties(self):
         result = generate_fit_result()
         self.assertEqual(result._parameters, [1.6, 0.6, 0.7])
         self.assertEqual(result._names, ['n', 'r', 'alpha'])
-        self.assertEqual(result.parameters, {'r':0.6, 'n':1.6, 'alpha':0.7})
-        self.assertEqual(result.scatterer,
-                          Sphere(n=1.6, r=0.6, center=[10, 10, 10]))
+        self.assertEqual(result.inferred_parameters,
+                         {'r':0.6, 'n':1.6, 'alpha':0.7})
+        self.assertEqual(result.inferred_scatterer,
+                         Sphere(n=1.6, r=0.6, center=[10, 10, 10]))
+
+    @attr("medium")
+    def test_inferred_hologram(self):
+        result = generate_fit_result()
+        self.assertAlmostEqual(
+            result.inferred_hologram.mean().item(), 1.005387, places=6)
+        self.assertTrue(hasattr(result, '_inferred_hologram'))
+
+    @attr("medium")
+    def test_best_fit_returns_inferred_hologram(self):
+        result = generate_fit_result()
+        with warnings.catch_warnings():
```

Comment:
This test is failing on Travis. I think you need to do:
```python
with warnings.catch_warnings():
    warnings.simplefilter('ignore')
    best_fit = result.best_fit()
```

Reply:
Should be fixed if Travis passes.